shiny/Data-Ingest/modules/formats_variables_module.R

```diff
@@ -0,0 +1,45 @@
+### UI ###
+formats.varsUI <- function(id) {
+  ns <- NS(id)
+  
+  box(
+    title = h2("3. Formats-Variables"), width = 4, solidHeader = TRUE, status = "success", collapsible = TRUE, collapsed = FALSE, 
+    selectizeInput(ns("pecan_var"), choices = NULL, label = "Variable",
+                   options = list(
+                     placeholder = 'Please search or select a site below',
+                     onInitialize = I('function() { this.setValue(""); }')
+                   )
+                   ),
+    splitLayout(
+      textInput(ns("var_name"), label = "Name"),
+      textInput(ns("var_unit"), label = "Unit")
+    ),
+    splitLayout(
+      textInput(ns("storage_type"), label = "Storage Type"),
+      textInput(ns("col_num"), label = "Column Number")
+    )
+  )
+}
+
+### Server ###
+formats.vars <- function(input, output, session){
+  
+  ##Output list 
+  FormatVars <- list()
+  
+  ## Machine Name ##
+  variables <- c("1","2","3","4","5")
+  updateSelectizeInput(session = getDefaultReactiveDomain(), "pecan_var", choices = variables, server = TRUE)
+  
+  observeEvent(input$createDBFilesRecord, {
```

Comment:
why does the format_vars module include the creation of the DBfiles record? Formats don't depend on machine ID or input paths or names so I'm not following what the code below does